### PR TITLE
Add a set attendance feature

### LINKED ISSUE #110: As a CCA organizer/exco member
I can track how often someone attends through a contact attendance feature make administration of the CCA easier.

### PR DESCRIPTION
**Pull request by chuahjiajie**
Closes #110 

Here's how it looks like, setting the attendance of the first user to 6/7:
image

There are still 4 testcases for storage that are failing.

### CODE REVIEW

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java [hunk 1]
```diff
@@ -141,12 +153,37 @@ public Person toModelType() throws IllegalValueException {
...
+            throw new IllegalValueException(Attendance.MESSAGE_LESS_THAN_CONSTRAINT);
+        }
         final Address modelAddress = new Address(address);
         final Set<Role> modelRoles = new HashSet<>(personRoles);
         final Set<Cca> modelCcas = new HashSet<>(personCcas);
 
-        return new Person(modelName, modelPhone, modelEmail, modelAddress, modelRoles,
-                modelCcas, modelAmount, modelMetadata);
+        return new Person(modelName, modelPhone, modelEmail, modelAddress, modelRoles, modelCcas, modelAmount,
+                modelAtt, modelSess, , modelMetadata);
```
Comment (alphajae11) [inline src/main/java/seedu/address/storage/JsonAdaptedPerson.java]: There is a empty parameter - you have to fix this error

Comment (chuahjiajie) [reply src/main/java/seedu/address/storage/JsonAdaptedPerson.java]: Thank you. Fixed

File: src/main/java/seedu/address/model/person/Person.java [hunk 2]
```diff
@@ -147,14 +162,17 @@ public boolean equals(Object other) {
                 && address.equals(otherPerson.address)
                 && roles.equals(otherPerson.roles)
                 && ccas.equals(otherPerson.ccas)
+                && otherPerson.getAmount().equals(getAmount())
+                && attendance.equals(otherPerson.attendance)
+                && sessions.equals(otherPerson.sessions);
```
Comment (alphajae11) [inline src/main/java/seedu/address/model/person/Person.java]: I think you have to get rid of semi-colon here

File: src/main/java/seedu/address/model/person/Person.java [hunk 3]
```diff
@@ -147,14 +162,17 @@ public boolean equals(Object other) {
                 && address.equals(otherPerson.address)
                 && roles.equals(otherPerson.roles)
                 && ccas.equals(otherPerson.ccas)
+                && otherPerson.getAmount().equals(getAmount())
```
Comment (alphajae11) [inline src/main/java/seedu/address/model/person/Person.java:165]: shouldn't this be amount.equals(otherPerson.getAmount())

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 4]
```diff
@@ -168,6 +170,34 @@ public static Set<Cca> parseCcas(Collection<String> ccas) throws ParseException
...
+     */
+    public static Sessions parseSess(String sessions) throws ParseException {
+        requireNonNull(sessions);
+        String trimmedSess = sessions.trim();
+        if (!Sessions.isValidSessions(trimmedSess)) {
+            throw new ParseException(Sessions.MESSAGE_CONSTRAINTS);
+        }
+        return new Sessions(sessions);
+
      * Parses {@code Metadata metadata} into a {@code Metadata}.
```
Comment (alphajae11) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:206]: I think you should comment this correctly - I think its causing error to the source code

File: src/main/java/seedu/address/logic/parser/SetAttCommandParser.java [hunk 5]
```diff
@@ -0,0 +1,63 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.*;
```
Comment (alphajae11) [inline src/main/java/seedu/address/logic/parser/SetAttCommandParser.java:5]: I think you should not use * for import statement in order to adhere to the code standards

Comment (alphajae11) [inline src/main/java/seedu/address/logic/parser/SetAttCommandParser.java:5]: I think you should avoid using * for import statement in order to follow the code standards

**Review by alphajae11 — COMMENTED** [review]
Overall, it looks okay - just a few minor errors that are probably causing the codecov to fail

**Review by JuliaPoo — APPROVED** [review]

**Pull request merged**